### ISSUE TITLE
negative loss value

### ISSUE THREAD

**Issue by shartoo** [npuichigo/waveglow#5]
i'm using your code to train my dataset,the codes ran ok while loss value is negative ,is this normal?
```
INFO:pytorch:Let's use 1 GPUs!
C:\Python35\lib\site-packages\torch\nn\modules\upsampling.py:122: UserWarning: nn.Upsampling is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.Upsampling is deprecated. Use nn.functional.interpolate instead.")
INFO:pytorch:[1,   1] loss: 8774.103
INFO:pytorch:[1,   2] loss: 247.321
INFO:pytorch:[1,   3] loss: 2267.624
INFO:pytorch:[1,   4] loss: 2223.914
INFO:pytorch:[1,   5] loss: 21230.785
INFO:pytorch:[1,   6] loss: -4516.562
INFO:pytorch:[1,   7] loss: -1636.356
INFO:pytorch:[1,   8] loss: -1802.424
INFO:pytorch:[1,   9] loss: -973.106
INFO:pytorch:[1,  10] loss: -2390.099
INFO:pytorch:[1,  11] loss: -3500.059
INFO:pytorch:[1,  12] loss: -2850.755
INFO:pytorch:[1,  13] loss: -4785.271
INFO:pytorch:[1,  14] loss: -5666.863
INFO:pytorch:[1,  15] loss: -6398.563
INFO:pytorch:[1,  16] loss: 6809.037
INFO:pytorch:[1,  17] loss: 6204.096
INFO:pytorch:[1,  18] loss: -9287.712
INFO:pytorch:[1,  19] loss: -3614.344
```

**Comment by npuichigo** [comment]
The value of a probability density function is not restricted to the range of 0 and 1, so -log(P(x)) may be negative.

**Comment by shartoo** [comment]
您好！我之前听过您在AI研习社的关于语音合成的演讲，做了一些[笔记](http://www.mooc.ai/course/600/learn?lessonid=2932#lesson/2932),有一个问题想请教下。传统方法合成的语音有很重的机械音，如果把声码器换成神经网络的，比如wavenet这种，会出现跳帧的情况。就是有些字的发音会没有，请问这种问题该如何解决呢？

**Comment by ajinkyakulkarni14** [comment]
Why the calculated loss in your implementation is different than the one explained in nvidia version of Waveglow ?
